frontend/src/tests/setup.js

```diff
@@ -0,0 +1,167 @@
+/**
+ * @jest-environment node
+ */
+import { expect } from "./utils";
+
+class DatabaseSeeder {
```

Comment:
dumb question: why did you use class? Can't we use pure functions?

Reply:
We need to store the `seeded` data so that we can export it. It could be a global variable instead...but a class also works. In this case, it's just a way to group related functions.

Reply:
I see, ty.